docs/guides/introduction-to-camunda-cloud.md

```diff
@@ -5,7 +5,14 @@ sidebar_label: Introduction to Camunda Cloud
 slug: /guides/
 ---
 
-Camunda Cloud delivers scalable, on-demand process automation as-a-service with powerful execution engines for BPMN processes and DMN decisions paired with tools for collaborative modeling, operations and analytics.
+:::note
+
+Looking for a Hands-on Intro? Sign up for [Camunda Cloud](https://camunda.io) to walkthrough our Getting Started Experience to see all the components of Camunda Cloud.
```

Comment:
I really like this addition! I think it's particularly great for anyone who hasn't seen the new GSE or happens to start with the documentation first for whatever reason. To confirm, if someone has already signed up for an account, they'll still be able to experience the GSE in the Console, correct?

Reply:
Yes! The fancy new homepage promotes this as the Product Walkthrough.